### PR TITLE
tests: Enable rpmdb-sqlite for EL9

### PR DESCRIPTION
**Pull request by travier**
From https://github.com/coreos/fedora-coreos-config/pull/2131/files#r1050151417

Will conflict with https://github.com/coreos/fedora-coreos-config/pull/2131 so I'll rebase when its merged.

### CODE REVIEW

File: tests/kola/files/rpmdb-sqlite [hunk 1]
```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
-# kola: { "distros": "fcos", "exclusive": false }
+# kola: { "exclusive": false }
 # This test only runs on FCOS as the RHCOS rpmdb has not migrated to sqlite yet.
 # TODO-RHCOS: drop the "fcos" tag when RHCOS migrates to using sqlite
```
Comment (dustymabe) [inline tests/kola/files/rpmdb-sqlite]: clean up comment or maybe move some of it down to the if statement below to add context.

Comment (travier) [reply tests/kola/files/rpmdb-sqlite]: Yes, this is waiting on the rebase for #2131

**Comment by travier** [comment]
Ready for review

**Comment by travier** [comment]
Manually tested passing for SCOS.

**Review by jlebon — APPROVED** [review]

**Review by dustymabe — APPROVED** [review]

**Pull request merged**